qutebrowser/config/configdata.py

```diff
@@ -301,6 +301,10 @@
     )),
 
     ('completion', sect.KeyValue(
+        ('download-path-suggestion',
+         SettingValue(typ.DownloadPath(), 'path'),
+         "What to show in the suggestion for the download question."),
```

Comment:
I think this wouldn't be clear to an user - what about *What to display in the download filename input* or so?